metabase/metabase
# 🥳 Miscellaneous Bugfixes 🥳

Resolves: all open bugs

# Description

Let me know if you see any that I missed!

## How to test

- checkout branch
- `npm run start`
- click around to make sure there are no bugs anymore
